src/Contact.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by brianmichael on 1/23/17.
+ */
+import React from 'react';
+import Highlighter from './Highlighter';
+
+const Contact = (props) => {
+
+  return (
+    <li
+      className="contact"
+      >
+      <div className="image-cropper">
+        <img src={props.avatar} alt="avatar" />
+      </div>
+      <div className="contact-info">
+        <h2>
+          <Highlighter searchWords={props.value} textToHighlight={props.name} />
+        </h2>
+        <h3>{props.occupation}</h3>
+      </div>
+
+      <button
+        className="select-button"
+        id={props.id}
+        onClick={(event) => props.handleSelectContactClick(event)}>Select
+        </button>
+    </li>
+  );
+};
+
+Contact.propTypes = {
+  avatar: React.PropTypes.string,
+  value: React.PropTypes.string,
+  name: React.PropTypes.string,
+  occupation: React.PropTypes.string,
+  id: React.PropTypes.number,
+  handleSelectContactClick: React.PropTypes.func
```

Comment:
You probably want to define those as `isRequired`. Otherwise you need to check everywhere in the component if that value is undefined before you use it. 

You can use `import {PropTypes} from 'react'` to simplify the code here.